nucypher/crypto/powers.py

```diff
@@ -92,37 +92,39 @@ class BlockchainPower(CryptoPowerUp):
     """
     not_found_error = NoBlockchainPower
 
-    def __init__(self, blockchain: 'Blockchain', account: str) -> None:
+    def __init__(self, client=None, device=None) -> None:
         """
         Instantiates a BlockchainPower for the given account id.
         """
-        self.blockchain = blockchain
-        self.account = account
+        self.device = device
+        self.client = client
         self.is_unlocked = False
 
-    def unlock_account(self, password: str):
+    def unlock_account(self, account, password: str):
         """
         Unlocks the account for the specified duration. If no duration is
         provided, it will remain unlocked indefinitely.
         """
-        self.is_unlocked = self.blockchain.interface.unlock_account(self.account, password)
+        self.is_unlocked = self.client.unlock_account(account, password)
```

Comment:
Wait, wasn't this meant to be account-agnostic? Why is then `self.is_unlocked` specific to this `account`?
Anyway, what I really want is to go back to account-specific BlockchainPower.

Reply:
Not the comment in the PR description: 

> Note: Calls to BlockchainPower or TransactingPower can be considered concept-code, for that, see PR #1092

Reply:
Resolved in 11e79d0 :heavy_check_mark: